docs/site/guides/rest/customize-openapi.md

```diff
@@ -0,0 +1,48 @@
+---
+lang: en
+title: 'Customizing how OpenAPI spec is served'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, OpenAPI, Customize
```

Comment:
Maybe `Customize` -> `Customization`? (it seems like our keywords are nouns, so might want to keep it consistent)